docs/features/common_functional_options.md

```diff
@@ -32,7 +128,7 @@ postgres, err = postgresModule.Run(ctx, "postgres:15-alpine", testcontainers.Wit
 If you need to access a port that is already running in the host, you can use `testcontainers.WithHostPortAccess` for example:
 
 ```golang
-postgres, err = postgresModule.Run(ctx, "postgres:15-alpine", testcontainers.WithHostPortAccess(8080))
+ctr, err = aModule.Run(ctx, "image:tag", testcontainers.WithHostPortAccess(8080))
```

Comment:
suggestion: use an idiomatic name, as I read that and thought it was a bug vs a placeholder.
More below